src/lib/streaming.ts

```diff
@@ -0,0 +1,55 @@
+import { PassThrough } from "stream";
```

Comment:
Nit: would you mind adding a docstring to this file explaining what it does and linking to https://aws.amazon.com/fr/blogs/compute/introducing-aws-lambda-response-streaming/ ? Thanks

Reply:
Good idea! Done in 4acb1772079f234036dc20dae339191936d79d8c.